### PR TITLE
Php cs fixer refactoring

### PR DESCRIPTION
**Pull request by raphaelstolt**
Keep Composer script utilisation consistent and enable 'php-cs-fixer' alpha version for repository.

### CODE REVIEW

**Comment by raphaelstolt** [comment]
Failing builds seems to be caused by Travis CI infrastructure problems.

File: tests/stubs/with-phpcs/travis.stub [hunk 1]
```diff
@@ -12,5 +12,5 @@ before_script:
   - travis_retry composer install --no-interaction
 
 script:
-  - vendor/bin/php-cs-fixer fix --diff --verbose --dry-run
+  - composer cs-lint
```
Comment (jonathantorres) [inline tests/stubs/with-phpcs/travis.stub:15]: Why are we adding the cs-lint on the stub files? I thought we were just gonna add the `2.0.0-alpha` release just on the construct codebase.

Comment (raphaelstolt) [reply tests/stubs/with-phpcs/travis.stub:15]: Linting is also possible with the pre-`2.0.0-alpha` and was already introduced with 1d74c2dc16f5903250bd7247b82215ba4303354b. What `2.0.0-alpha` adds is a way to cache the linting process for Travis CI builds. The constructed projects will all have a pre-`2.0.0-alpha` dependency.

Comment (jonathantorres) [reply tests/stubs/with-phpcs/travis.stub:15]: Got it 👍

**Pull request merged**